src/pool.ts

```diff
@@ -301,8 +319,13 @@ export class Pool {
     this.liquidityAsset = liquidityAsset;
     this.internalState = internalState;
 
-    if (internalState.INITIAL_A !== undefined) {
-      this.poolType = "STABLESWAP";
+    this.poolType = getPoolTypeFromInternalState(internalState);
+
+    if (this.poolType === "CONSTANT_PRODUCT") {
+      this.params = {
+        feeBps: internalState.FEE_BPS,
```

Comment:
Stableswap in this PR still has `pactFeeBps`. Is it on purpose that constant product doesn't have it (as we were talking it this pact fee bps is not relevant info for sdk user)

Reply:
Added, missed that.